src/pages/index.js

```diff
@@ -1,118 +1,273 @@
-const addCardPopup = new PopupWithForm({
-  popupSelector: "#add-card-modal",
-  handleFormSubmit: (inputValues) => {
-    const newCard = createCard(inputValues);
-    cardList.prependItem(newCard);
-    addCardPopup.close();
-  },
-  loadingButtonText: "Saving...",
-});
-addCardPopup.setEventListener();
-profileAddButton.addEventListener("click", () => {
-  addFormValidator.resetValidation(); // Clear validation errors before opening
-  addCardPopup.open();
-});
+import Card from "../components/Card.js";
+import FormValidator from "../components/FormValidator.js";
+import Section from "../components/Section.js";
+import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithForm from "../components/PopupWithForm.js";
+import UserInfo from "../components/UserInfo.js";
+import "../pages/index.css";
+import {
+  // initialCards,
+  profileEditButton,
+  profileAddButton,
+  profileEditModal,
+  addCardModal,
+  previewImageModal,
+  profileModalCloseButton,
+  addCardModalCloseButton,
+  previewModalCloseButton,
+  profileTitle,
+  profileDescription,
+  modalTitleInput,
+  modalDescriptionInput,
+  profileEditForm,
+  addCardForm,
+  cardListEl,
+  cardTemplate,
+  cardTitleInput,
+  cardUrlInput,
+  previewImage,
+  previewFooter,
+  formValidationConfig,
+} from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
+
+// =============================================================================
+// API + userInfo
+// =============================================================================
+
+let userID;
+let cardList;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "bcc6c959-bae4-4de1-885c-059746718b41",
+    "Content-Type": "application/json",
+  },
+});
+
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  professionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([user, initialCards]) => {
+    userInfo.setUserInfo(user);
+    userID = user._id;
+    userInfo.setUserInfo(user);
+
+    cardList = new Section(
+      {
+        items: initialCards,
+        renderer: ({ name, link, isLiked, _id, userId, ownerId }) => {
+          const newCard = createCard({
+            name,
+            link,
+            isLiked,
+            _id,
+            userId,
+            ownerId,
+          });
+          cardList.addItem(newCard);
+        },
+      },
+      ".cards__list"
+    );
+    cardList.renderItems();
+  })
+  .catch(console.error);
+
+// =============================================================================
+// Handle Clicks
+// =============================================================================
+
+function handleLikeClick(cardInstance) {
+  if (cardInstance._likes) {
```

Comment:
`_likes`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.